xc/common/cmake/project_ray.cmake

```diff
@@ -139,6 +143,7 @@ function(PROJECT_RAY_ARCH)
     )
 
   add_file_target(FILE channels.db GENERATED)
+  add_file_target(FILE vpr_grid_map.csv GENERATED)
```

Comment:
Need to linearize these outputs.  Make the target of `vpr_grid_map.csv` depend on the target of `channels.db`.

Reply:
Done